src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/fonts.fluent.scss

```diff
@@ -1,6 +1,6 @@
 ﻿@font-face {
     font-family: 'Fabric MDL2 Assets';
-    src: url('../assets/fluent/fonts/FabExMDL2.3.36.ttf') format("truetype");
+    src: url('../assets/fluent/fonts/fabric-icons.woff') format("woff");
```

Comment:
add the version of the fabric icons to the file name.
the current version of the font is visible on the UI fabric font website (https://uifabricicons.azurewebsites.net/) as shown in the following image:
![image](https://user-images.githubusercontent.com/6169846/164455830-14f88cb6-1de6-43ed-a72d-15c2f6151916.png)

Reply:
use a filename like this: `FabMDL2.v4.46.woff`